bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -31,6 +31,7 @@
 import org.openjdk.skara.host.HostUser;
 import org.openjdk.skara.issuetracker.Comment;
 import org.openjdk.skara.issuetracker.Issue;
+import org.openjdk.skara.issuetracker.IssueTrackerIssue;
 import org.openjdk.skara.json.JSONValue;
```

Comment:
`import org.openjdk.skara.issuetracker.Issue;` and `import org.openjdk.skara.json.JSONValue;` can be removed